iOSClient/Utility/NCContentPresenter.swift

```diff
@@ -60,63 +60,56 @@ class NCContentPresenter: NSObject {
         case info
     }
     
-    @objc private var lastErrorCode: Int = 0
-
     //MARK: - Message
     
-    @objc func messageNotification(_ title: String, description: String?, delay: TimeInterval, type: messageType, errorCode: Int, forced: Bool = false) {
+    @objc func messageNotification(_ title: String, description: String?, delay: TimeInterval, type: messageType, errorCode: Int) {
+        messageNotification(title, description: description, delay: delay, type: type, errorCode: errorCode, priority: .normal, dropEnqueuedEntries: false)
+    }
+    
+    func messageNotification(_ title: String, description: String?, delay: TimeInterval, type: messageType, errorCode: Int, priority: EKAttributes.Precedence.Priority = .normal, dropEnqueuedEntries: Bool = false) {
                        
-        // No notification message
-        if forced == false {
-            
-            if errorCode == -999 { return }         // Cancelled transfer
-            else if errorCode == 200 { return }     // Transfer stopped
-            else if errorCode == 207 { return }     // WebDAV multistatus
-            else if errorCode == 423 { return }     // WebDAV locked
-            else if errorCode == -1001 { return }   // Time out
-            else if errorCode == -1005 { return }   // Connection lost
-            else if errorCode == 0 && type == messageType.error { return }
-            
-            // No repeat message for:
-            if errorCode == lastErrorCode {
-                if errorCode ==  Int(CFNetworkErrors.cfurlErrorNotConnectedToInternet.rawValue) { return }
-            } else {
-                lastErrorCode = errorCode
-            }
-        }
+        // No notification message for:
+        if errorCode == -999 { return }         // Cancelled transfer
+        else if errorCode == 200 { return }     // Transfer stopped
+        else if errorCode == 207 { return }     // WebDAV multistatus
+        //else if errorCode == 423 { return }     // WebDAV locked
+        //else if errorCode == -1001 { return }   // Time out
+        //else if errorCode == -1005 { return }   // Connection lost
```

Comment:
Are those still needed? if not remove to avoid clutter..

Reply:
yes the -999 is not a error message as 200 and 207

Reply:
I was talking about the commented lines (75-77). If they are needed for clarity add it to the function's documentation. Otherwise please remove them.